libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/SpecTranslate/Conway/Base.hs

```diff
@@ -530,48 +630,95 @@ instance SpecTranslate ctx (DRep c) where
   toSpecRep DRepAlwaysAbstain = pure Agda.AbstainRep
   toSpecRep DRepAlwaysNoConfidence = pure Agda.NoConfidenceRep
 
+instance SpecTranslate ctx Url where
+  type SpecRep Url = T.Text
+  toSpecRep = pure . urlToText
+
 instance SpecTranslate ctx (Anchor c) where
   type SpecRep (Anchor c) = Agda.Anchor
-  toSpecRep _ = pure ()
+  toSpecRep (Anchor url _) = Agda.Anchor <$> toSpecRep url <*> pure ()
 
 instance SpecTranslate ctx (TxId era) where
   type SpecRep (TxId era) = Agda.TxId
 
-  toSpecRep (TxId x) = Agda.MkTxId <$> toSpecRep x
+  toSpecRep (TxId x) = toSpecRep x
 
-toAgdaTxBody ::
-  ( SpecRep (TxOut era) ~ Agda.TxOut
-  , SpecRep (TxCert era) ~ Agda.TxCert
-  , EraTx era
-  , BabbageEraTxBody era
+instance SpecTranslate ctx (Withdrawals era) where
+  type SpecRep (Withdrawals era) = Agda.Wdrl
+
+  toSpecRep (Withdrawals w) = toSpecRep w
+
+instance SpecTranslate ctx (AuxiliaryDataHash c) where
+  type SpecRep (AuxiliaryDataHash c) = ()
+
+  toSpecRep _ = pure ()
+
+instance
+  ( ConwayEraTxBody era
+  , TxBody era ~ ConwayTxBody era
+  , SpecRep (TxOut era) ~ Agda.TxOut
+  , SpecRep (ConwayTxCert era) ~ Agda.DCert
+  , SpecRep (PParamsHKD StrictMaybe era) ~ Agda.PParamsUpdate
+  , TxCert era ~ ConwayTxCert era
+  , Share (TxOut era) ~ Interns (Credential Staking (EraCrypto era))
```

Comment:
CI is failing
```suggestion
  , Share (TxOut era) ~ Interns (Credential 'Staking (EraCrypto era))
```

Reply:
Oh, interesting, I didn't know that ticks became optional in `9.6.6`